unitTests/ui/MLFacet/MLFacetValues/MLFacetValuesTest.ts

```diff
@@ -82,5 +83,32 @@ export function MLFacetValuesTest() {
       const element = mLFacetValues.render();
       expect(element.childElementCount).toBe(mockFacetValues.length);
     });
+
+    it(`when moreValuesAvailable is false
+      should not render the "Show more" button`, () => {
+      const element = mLFacetValues.render();
```

Comment:
Smaller point, but I like to reduce repetition by creating functions to retrieve elements. In this case, you could use functions to get the more and less buttons. e.g.

```
function moreButton() {
    const element = mLFacetValues.render();
    return $$(element).find('.coveo-ml-facet-show-more');
}
```

Reply:
Nice